.github/scripts/mr_publish_results.py

```diff
@@ -18,17 +18,17 @@
 DD_SERVICE = "rasa"
 METRIC_RUNTIME_PREFIX = "rasa.perf.benchmark."
 METRIC_ML_PREFIX = "rasa.perf.ml."
-IS_EXTERNAL = os.environ["IS_EXTERNAL"]
-DATASET_REPOSITORY_BRANCH = os.environ["DATASET_REPOSITORY_BRANCH"]
+IS_EXTERNAL = os.getenv("IS_EXTERNAL")
```

Comment:
isn't `getenv` a bit risky as it hides the case when the env variable for some reason isn't filled? Or are they all optional? The old approach had the advantage that it would fail early in this case